resources/sphinx/sphinx/conf.py.in

```diff
@@ -232,45 +233,41 @@ todo_include_todos = True
 
 
 # cmake parser custom module
-class IfMode(Directive):
-
-    has_content = True
-    required_arguments = 1
-    optional_arguments = 0
-    final_argument_whitespace = True
-    option_spec = {}
-
-    def run(self):
-        cm = self.state.document.settings.env.config.currentmode
-        if cm == self.arguments[0]:
-            node = nodes.Element()
-            node.document = self.state.document
-            set_source_info(self, node)
-            self.state.nested_parse(self.content, self.content_offset,
-                                    node, match_titles=1)
-            return node.children
-        return []
-
-# cmake parser custom module
-
-
-class SetMode (Directive):
-    has_content = False
-    required_arguments = 1
-    optional_arguments = 0
-    final_argument_whitespace = False
-
-    def run(self):
-        self.state.document.settings.env.config.currentmode = self.arguments[0]
-        return []
+# class IfMode(Directive):
+#     has_content = True
+#     required_arguments = 1
+#     optional_arguments = 0
+#     final_argument_whitespace = True
+#     option_spec = {}
+
+#     def run(self):
+#         cm = self.state.document.settings.env.config.currentmode
+#         if cm == self.arguments[0]:
+#             node = nodes.Element()
+#             node.document = self.state.document
+#             set_source_info(self, node)
+#             self.state.nested_parse(self.content, self.content_offset,
+#                                     node, match_titles=1)
+#             return node.children
+#         return []
+
+# class SetMode (Directive):
+#     has_content = False
+#     required_arguments = 1
+#     optional_arguments = 0
+#     final_argument_whitespace = False
+
+#     def run(self):
+#         self.state.document.settings.env.config.currentmode = self.arguments[0]
+#         return []
```

Comment:
should we keep this ?

Reply:
I forgot I wanted to add the cmake support a bit better in this pr too. But yes it will be removed at a certain point but for now I may need it.